src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/ResiliencyReportData.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class ResiliencyReportData : IResiliencyReportData
+    {
+        [JsonConverter(typeof(ResiliencyResource))]
+        ResiliencyResource[] resiliencyResourceList;
+
+        /// <summary>
+        /// Creates an instance of ResiliencyReportData
+        /// </summary>
+        public ResiliencyReportData(string customerName, ResiliencyResource[] resiliencyResourceList)
+        {
+            this.CustomerName = customerName;
+            if (resiliencyResourceList == null)
```

Comment:
What about an empty list?

Reply:
Thanks. Changed it to this:

            if (resiliencyResourceList == null ||  resiliencyResourceList.Length == 0)